components/mpas-ocean/src/shared/mpas_ocn_vel_tidal_potential.F

```diff
@@ -169,9 +170,10 @@ subroutine ocn_vel_tidal_potential_tend(ssh, tend, err)!{{{
          kMax = maxLevelEdgeTop(iEdge)
 
          potentialGrad = - gravity*invdcEdge* &
-                         (tidalPotEta(cell2) - tidalPotEta(cell1) + &
-                          betaSelfAttrLoad* &
-                          (ssh(cell2) - ssh(cell1)))
+            (  tidalPotEta(cell2) - tidalPotEta(cell1) &
+             + betaSelfAttrLoad &
+              *((ssh(cell2) - ssh(cell1)) &
+                + rho0gInv*(surfacePressure(cell2) - surfacePressure(cell1))))
```

Comment:
@nairita87 I altered the spacing here and added `rho0gInv`. Please double check this.